src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasPointAndLineSlicer.java

```diff
@@ -407,4 +415,28 @@ private void updateLineShapePoints(final Line line)
         }
     }
 
+    /**
+     * For {@link Line}s that could not be cut (because of too many created points or too many
+     * created line segments), we will gather the country codes for all the slices and assign the
+     * multiple-country code value to the un-sliced line. As a result, the same un-sliced line will
+     * appear in Atlas files for all spanning countries.
+     *
+     * @param line
+     *            The {@link Line} in question
+     * @param slices
+     *            The {@link Geometry} slices for the given {@link Line}
+     */
+    private void updateLineToHaveCountryCodesFromAllSlices(final Line line,
+            final List<Geometry> slices)
+    {
+        final Map<String, String> tags = new HashMap<>();
+        final Set<String> allCountries = new HashSet<>();
+        slices.forEach(geometry -> allCountries
```

Comment:
This could result in varied ordering of the isoCountryCodes which could result in different values for ```IsoCountryTag.first```.  Do we want to consider enforcing some order here?

Reply:
Good catch, updated!